services/catalog/src/simcore_service_catalog/service/catalog_services.py

```diff
@@ -225,19 +227,14 @@ async def get_catalog_service(
     service_version: ServiceVersion,
 ) -> ServiceGetV2:
 
-    access_rights = await repo.get_service_access_rights(
-        key=service_key,
-        version=service_version,
+    access_rights = await check_catalog_service_permissions(
+        repo=repo,
         product_name=product_name,
+        user_id=user_id,
+        service_key=service_key,
+        service_version=service_version,
+        permission="read",
```

Comment:
enum?

Reply:
I find `Literal`s more handy in cases like this